src/Catrobat/AppBundle/Controller/Api/SearchController.php

```diff
@@ -21,6 +21,12 @@ public function searchProgramsAction(Request $request)
   {
     $program_manager = $this->get('programmanager');
     $query = $request->query->get('q');
+
+    $query = str_replace("yahoo", "", $query);
+    $query = str_replace("gmail", "", $query);
+    $query = str_replace("gmx", "", $query);
+    $query = trim($query);
```

Comment:
There would have been a way to avoid triming the query, but I don't expect it to cause any troubles so we will accept this solution.